schema/va-spec/aac-2017/diagnostic-study-statement-profile-source.yaml

```diff
@@ -28,3 +28,50 @@ $defs:
             A proposition about a diagnostic association between a variant and condition, for which the study provides
             evidence. The validity of this proposition, and the level of confidence/evidence supporting it,
             may be assessed and reported by the Statement.
+        strength:
+          description: >-
+            The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
+            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
+            Guidelines. Note that the enumerated value set here is bound to the ‘code’ field of the Coding object that
+            is nested inside a MappableConcept's primary coding. Conditional requirement: Strength is evaluated relative to
+            the direction indicated by the 'direction' attribute. If direction is either 'supports' or 'disputes', the
+            'strength' attribute is required.  If the direction is 'neutral', then the 'strength' attribute is not allowed.
+          properties:
+            primaryCoding:
+              code:
+                enum:              # TO DO: Kori/Alex to populate value set here, as appropriate
+                  -                
+                  -  
+                  -
+              system:
+                const: AMP/ASCO/CAP (AAC) Guidleines for the Interpretation and Reporting of Sequence Variants in Cancer, 2017
```

Comment:
Hm. I'm not sure about an enum here...

For CIViC, we use:

```yaml
- e000000
- e000001
- e000002
- e000003
- e000004
- e000005
- e000006
- e000007
- e000008
- e000009
- e000010
```

Which follows https://go.osu.edu/evidence-codes